hspy_bundle/configure_installer.py

```diff
@@ -161,10 +168,8 @@ def create_installers(self):
                         fa.write(line.replace("__NSIS_PLUGINS__",
                                               get_nsis_plugins_path()))
                     elif "__HSPY_ICON__" in line:
-                        icons = self.get_full_paths(
-                            "python-*\\Lib\\site-packages\\hyperspy\\data\\"
-                            "hyperspy_bundle_installer.ico", a)
-                        fa.write(line.replace("__HSPY_ICON__", icons))
+                        fa.write(line.replace("__HSPY_ICON__",
+                                              get_icon_path()))
```

Comment:
Couldn't we simply just copy the icon in `copy_files()` as well, and directly replace `__HSPY_ICON__` with `hyperspy_bundle_installer.ico` in the NSIS script?